fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -404,9 +403,7 @@ public void updateProperties(final IdentifierConverter<Resource, FedoraResource>
         final Model maybeModel = idTranslator.reverse().convert(this).getModel();
         final Model context = maybeModel == null ? createDefaultModel() : maybeModel;
 
-        final Deskolemizer deskolemize = new Deskolemizer(idTranslator, context);
-
-        final Model model = originalTriples.withThisContext(originalTriples.transform(deskolemize)).asModel();
+        final Model model = originalTriples.asModel();
```

Comment:
If you make this change, the update SPARQL will be running against RDF with skolem nodes in it. Attempts to deal with bnodes will always fail.